drivers/usb/device/Kconfig

```diff
@@ -229,6 +229,17 @@ config USB_NATIVE_POSIX
 	help
 	  Native Posix USB Device Controller Driver.
 
+DT_ZEPHYR_UDC0 := $(dt_nodelabel_path,zephyr_udc0)
+DT_ZEPHYR_UDC0_HS_SPEED := $(dt_node_str_prop_equals,$(DT_ZEPHYR_UDC0),maximum-speed,high-speed)
```

Comment:
I am hesitant about this because `zephyr_udc0` is for the new USB stack while you use the result for the old USB stack.

Reply:
No? That node is used in what you call the "old" stack (I'd prefer the "current" stack, as the "new" stack is still experimental).

Reply:
Can you point where in the code it is being used for things that run with "old" stack? Either my search omits these uses, I misunderstand something or there is no such use.

Reply:
Not sure what you mean. This is the nodelabel used by all boards (regardless of which stack, they are using).

See e.g. https://github.com/zephyrproject-rtos/zephyr/blob/0a3f2f0397a86425cc7d12fa3a0c0ab8020d80e1/boards/native/native_sim/native_sim.dts#L176 and https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/subsys/usb/cdc_acm/app.overlay

Reply:
They are not enablers for the new stack. This node is where you specify the child nodes for the classes. This naming was introduced in e4f894788a9f073814f2f0c81ea5343c640f5905.